packages/business-rules/src/rules/appeal/deadline-date.js

```diff
@@ -1,29 +1,52 @@
 const { add, endOfDay, isValid: isDateValid } = require('date-fns');
 const { appeal } = require('../../config');
 const isValid = require('../../validation/appeal/type/is-valid');
-const { APPEAL_ID } = require('../../constants');
+const isValidApplicationDecision = require('../../validation/appeal/application-decision/is-valid');
+const { APPEAL_ID, APPLICATION_DECISION } = require('../../constants');
 const BusinessRulesError = require('../../lib/business-rules-error');
 
 /**
  * @description Given an appeal's decision date, and an expiry period, determine the appeal's
  * deadline date.
  *
  * @param {Date} decisionDate
- * @param {Number} expiryPeriodInWeeks a positive number
+ * @param {string} appealType
+ * @param {string} applicationDecision
  * @returns {Date}
  *
  * @throws {BusinessRulesError}
  */
-module.exports = (decisionDate, appealType = APPEAL_ID.HOUSEHOLDER) => {
+module.exports = (
+  decisionDate,
+  appealType = APPEAL_ID.HOUSEHOLDER,
+  applicationDecision = APPLICATION_DECISION.REFUSED,
+) => {
+  const type = appealType === null ? APPEAL_ID.HOUSEHOLDER : appealType;
+  const decision =
+    applicationDecision === null ? APPLICATION_DECISION.REFUSED : applicationDecision;
```

Comment:
The default value is set only if its not passed. Since the model is set with null value, the default value is not set and I had to explicitly set it like this.

Reply:
Ok